entity-service-api/src/main/proto/org/hypertrace/entity/type/service/v2/entity_type_service.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+package org.hypertrace.entity.type.service.v2;
+
+message EntityType {
+  string name = 1;
+  string attribute_scope = 2;
+  string id_attribute_key = 3;
```

Comment:
I think we need name_attribute_key as well, given the current entity design which has a first class name field.

Reply:
Done